app/projects/page.jsx

```diff
@@ -171,7 +171,6 @@ const ProjectsPage = () => {
           <CustomButton
             key={label}
             style={{
-              color: theme.palette.mode === "light" ? "black" : "white",
```

Comment:
Please check it in both light and dark mode. I guess I've to add this due to this breaking in one of the theme